.github/workflows/Integrations.yml

```
@@ -5,50 +5,53 @@ on:
     tags: [v*]
   pull_request:
 
-concurrency:
-  # Skip intermediate builds: always.
-  # Cancel intermediate builds: only if it is a pull request build.
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
-
 jobs:
   test:
-    name: ${{ matrix.package.repo }}
+    name: ${{ matrix.package.repo }}/${{ matrix.package.group }}/${{ matrix.julia-version }}
     runs-on: ${{ matrix.os }}
+    env:
+      GROUP: GDAL
     strategy:
       fail-fast: false
       matrix:
         julia-version: [1]
         os: [ubuntu-latest]
         package:
-          - {user: yeesian, repo: ArchGDAL.jl}
+          - {user: yeesian, repo: ArchGDAL.jl, group: GDAL}
+          - {user: JuliaGeo, repo: GADM.jl, group: GDAL}
+          - {user: evetion, repo: GeoDataFrames.jl, group: GDAL}
+          - {user: rafaqz, repo: Rasters.jl, group: GDAL}
 
     steps:
       - uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@v1
         with:
           version: ${{ matrix.julia-version }}
           arch: x64
-      - uses: julia-actions/julia-buildpkg@v1
+      - uses: julia-actions/julia-buildpkg@latest
```

```suggestion
      - uses: julia-actions/julia-buildpkg@v1
```

Let's specify the version number like the other actions. If there are newer versions dependabot will let us know and we can upgrade on our own terms, see for instance #159.